docker-compose/docker-whisk-controller.env

```diff
@@ -36,7 +36,7 @@ LIMITS_ACTIONS_SEQUENCE_MAXLENGTH=20
 
 CONFIG_whisk_loadbalancer_blackboxFraction=0.1
 CONFIG_kamon_statsd_port=8125
-CONFIG_whisk_loadbalancer_invokerBusyThreshold=16
+CONFIG_whisk_loadbalancer_invokerUserMemory=1024 m
```

Comment:
Might have problems with space but not sure if it needs to be "1024 m"

Reply:
The quotes make it worse actually. 1024m works for some of the instances. Locally I still have the invoker crashing though.